docs/r2dbc.md

```diff
@@ -149,31 +149,34 @@ to `true` and `user`
 to the email address associated with your IAM user.
 
 You must shorten the full IAM user email into a database username. Due to
-different constraints on allowed characters in the database username, Mysql and
+different constraints on allowed characters in the database username, MySQL and
 postgres differ in how they shorten an IAM email address into a database
 username.
 
-* Mysql: Truncate the IAM email removing the `@` and everything that follows.
+* MySQL: Truncate the IAM email removing the `@` and everything that follows.
 * Postgres: If the IAM email ends with `.gserviceaccount.com`, remove
   the `.gserviceaccount.com` suffix from the email.
 
-For example, if the full IAM user account
-is `my-sa@my-project.iam.gserviceaccount.com`,
-You would update the code, replacing these examples with the appropriate values
-for your database engine:
-
-|               | Mysql | Postgres             |
-|---------------|-------|----------------------|
-| <PROTOCOL>    | mysql | postgresql           |  
-| <IAM_DB_USER> | my-sa | my-sa@my-project.iam |
+For example, if the full IAM user account is 
+`my-sa@my-project.iam.gserviceaccount.com`, then the shortened database username
+would be `my-sa` for MySQL, and `my-sa@my-project.iam` for Postgres. 
 
 **Note:** a non-empty string value for the `password` property must be set.
 While this property will be ignored when connecting with the Cloud SQL Connector
 using IAM auth, leaving it empty will cause driver-level validations to fail.
 
+#### Example
+
+Replace these parameters in the example based on your database type:
+
+|          | PROTOCOL   | IAM_DB_USER          |
```

Comment:
Let's add a header here in that case: "DB" or "DB Engine"

Reply:
Same thing elsewhere.

Reply:
Fixed. I used "Database" so that it is not confused as a parameter, which are formatted all-caps.